### PR TITLE
Begin tidying/improving code

### PR DESCRIPTION
**Pull request by windymilla**
Ensure every sub has at least a one line comment describing its purpose
Remove superfluous Convert Windows CP1252 Characters option

### CODE REVIEW

**Comment by windymilla** [comment]
Since I've edited pretty much every Perl file, I thought I'd get this merged before beginning to tackle the many issues I discovered while doing this.

File: src/lib/Guiguts/MultiLingual.pm [hunk 1]
```diff
@@ -761,64 +780,8 @@ sub addminfreqproject {
     getwordcounts();
 }
 
-sub lowergetmisspelled {
```
Comment (cpeel) [inline src/lib/Guiguts/MultiLingual.pm:764]: This was the only edit I saw that wasn't immediately obvious. I assume this was an unused function?

**Review by cpeel — APPROVED** [review]
I skimmed all of the changes (I did not read every function comment you added). All of the deletions / cleanups made sense to me except one which I wanted to highlight.

Yay for much cleaner code!

**Review by srjfoo — APPROVED** [review]

File: src/lib/Guiguts/MultiLingual.pm (hunk 1 above)
Comment (windymilla) [reply src/lib/Guiguts/MultiLingual.pm:764]: Yes - not referenced anywhere else in the code

**Pull request merged**